### ISSUE TITLE
How to find only company name but not ot agencies, institutions, etc in "ORG" ? 

### ISSUE THREAD

**Issue by RocheR2D2** [explosion/spaCy#3779]
i want to find only company name but not others in the entity type "ORG", anyone can help plz ?

**Comment by ines** [comment]
The pre-trained English models were traind on ~2 million words where the label `ORG` was defined as "companies, organisations, institutions" etc. So there's no easy fix to just "change that". You can either train your own model on data that's annotated with *your* interpretation of the label `ORG`, try to fine-tune the model (tricky because you also need a lot of data), or add rules on top.

See here for the relevant docs:
* https://spacy.io/usage/training
* https://spacy.io/usage/rule-based-matching#models-rules
